src/main/java/org/zalando/nakadi/util/AuthorizationUtils.java

```diff
@@ -55,7 +56,7 @@ public static void authorizeSubscriptionRead(final EventTypeRepository eventType
                             authorizeStreamRead(authorizationService, client, eventType);
                         });
                     } catch (final InternalNakadiException e) {
-                        e.printStackTrace();
+                        throw new MyNakadiRuntimeException1("Failed authorizing");
```

Comment:
As we agreed not to use MyNakadiRuntimeException1 for throwing, I suggest ServiceTemporarilyUnavailableException